Blazor/Layouts/CDTSLayout.razor

```diff
@@ -0,0 +1,38 @@
+﻿@inherits LayoutComponentBase
+
+    <GoC.WebTemplate.Blazor.Utils.HandleLanguageLink></GoC.WebTemplate.Blazor.Utils.HandleLanguageLink>
+
+    <Top TopObj = "@Top"/>
+
+    @if(@Top.TopSecMenu)
+    {
+        <SectionMenu LeftMenuItems = "@LeftMenu"/>
+    }
+    <main id="cdts-main" property="mainContentOfPage" typeof="WebPageElement" class="@(@Top.TopSecMenu? "col-md-9" : "container")">
+        @ChildContent
+        <PreFooter PreFooterObj = "@PreFooter"/>
+    </main>
+
+    <Footer FooterObj = "@Footer"/>
+
+@code{
+    private GoC.WebTemplate.Components.Entities.ITop _top;
```

Comment:
This variable is no longer used (The only warning left that I could see when I built the app)

Reply:
Removed